### PR TITLE
[multi-asic]: Update port_config files for multi-asic vs image

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
**- Why I did it**
Modify port_config.ini files to match with the changes done for multi-asic platform.

**- How I did it**
Added new columns: index, asic_port_name, role(Internal/external)
Modify alias of interface names. Alias should match the interface names present in minigraph file.

**- How to verify it**
Should be able to bring up multi-asic VS image.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [X] 201911
- [X] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+Ethernet32    13,14,15,16      Ethernet1/9    8       Eth8-ASIC0        E     40000
+Ethernet36    9,10,11,12       Ethernet1/10   9       Eth9-ASIC0        E     40000
+Ethernet40    17,18,19,20      Ethernet1/11   10      Eth10-ASIC0       E     40000
+Ethernet44    21,22,23,24      Ethernet1/12   11      Eth11-ASIC0       E     40000
+Ethernet48    53,54,55,56      Ethernet1/13   12      Eth12-ASIC0       E     40000
+Ethernet52    49,50,51,52      Ethernet1/14   13      Eth13-ASIC0       E     40000
+Ethernet56    57,58,59,60      Ethernet1/15   14      Eth14-ASIC0       E     40000
+Ethernet60    61,62,63,64      Ethernet1/16   15      Eth15-ASIC0       E     40000
+Ethernet64    69,70,71,72      Eth16-ASIC0    0       Eth16-ASIC0       I     40000
+Ethernet68    65,66,67,68      Eth17-ASIC0    1       Eth17-ASIC0       I     40000
```
Comment (judyjoseph) [inline device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini:19]: Backplane interface here for ASIC0 ?

Comment (SuvarnaMeenakshi) [reply device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini]: Fixed the interface names.

Comment (lguohan) [reply device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini]: do we want to have "Ext", "Int", wonder if in the future we will have more role/type? for example, fabric link.

Comment (SuvarnaMeenakshi) [reply device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini]: Updated as per suggestion.

**Pull request merged**